IPython/lib/display.py

```diff
@@ -263,13 +263,15 @@ class IFrame(object):
             src="{src}{params}"
             frameborder="0"
             allowfullscreen
+            {extra}
         ></iframe>
         """
 
-    def __init__(self, src, width, height, **kwargs):
+    def __init__(self, src, width, height, extra="", **kwargs):
```

Comment:
Minor thing: it would look better to use an `extras: List = None` here, and append the `.join()`-ed items to the iframe text.

Constraining this to only one extra field seems a bit too limitting in such generic class.

Reply:
Thank you for reviewing.

Because attributes are separated by spaces, `str` also allows to set multiple fields. I think different users will have different preferences, as shown below:
```python
# My assumption.
extra = '''
    example="1"
    example2="2"
'''

extra = [
    'example="1"',
    'example="2"'
]

extra = {
    "example1": "1",
    "example2": "2",
}
# There are some attributes without value, so dict seems to be hard to use.
```
Should `extra` support both str and list?

Reply:
To me it's more natural and Pythonic to use an ordered collection (note the "s" at the end of `extras`):

```
extras = ['example="1"', 'example="2"']
```

That way the fact that it can take attributes - plural - isn't hidden behind a singular name.